apps/web/src/utils/moderation.ts

```
@@ -23,56 +23,36 @@ async function moderateContentWithAI(text: string) {
-  // Check for international phone number patterns with various separators
-  const internationalPhonePattern = /\+\d[\d\s\-\.]{5,14}/;
+  // Email address pattern
+  const emailPattern = /[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}/i;
+
+  // Comprehensive phone number pattern covering various formats
+  const phoneNumberPattern =
+    /(\+?\d{1,3}[-.\s]?)?(\d{3}[-.\s]?)?\d{3}[-.\s]?\d{4}|\+\d[\d\s\-\.]{5,14}|\+\d{6,15}/i;
 
+  // Detect suspicious Unicode patterns
+  const suspiciousUnicodePattern =
+    /[\u2028-\u202F\u2800-\u28FF\u3164\uFFA0\u115F\u1160\u3000\u2000-\u200F\u2028-\u202F\u205F-\u206F\uFEFF\uDB40\uDC20\uDB40\uDC21\uDB40\uDC22\uDB40\uDC23\uDB40\uDC24]/;
```

Biome (1.9.4)

[error] A character class cannot match a surrogate pair. Add the 'u' unicode flag to match against them.

A surrogate pair forms a single codepoint, but is encoded as a pair of two characters. Without the unicode flag, the regex matches a single surrogate character.
Safe fix: Add unicode u flag to regex


(lint/suspicious/noMisleadingCharacterClass)